### ISSUE TITLE
Notify Client Exception is not a standard json object

### ISSUE THREAD

**Issue by CodrinCotarlanSoftwire** [alphagov/notifications-net-client#161]
https://github.com/alphagov/notifications-net-client/blob/358238c2f5a1c466c7704d8ac0749bc9dd2b1ce3/src/GovukNotify/Client/BaseClient.cs#L118-L129

When an exception occurs, a NotifyClientException is thrown, but its content is a plain string as outlined above. The string contains the information from the [documentation](https://docs.notifications.service.gov.uk/net.html#send-an-email-error-codes), but in a very unintuitive way, making it difficult to parse. For example, here is the string from the 'message' field one one such case:

`Status code 400. Error: {"errors":[{"error":"ValidationError","message":"email_address Not a valid email address"}],"status_code":400}, Exception: Status code 400. The following errors occured [{"error": "ValidationError", "message": "email_address Not a valid email address"}]`

I would much rather work with just the following, since it is much easier to parse, or maybe even incorporate those as fields in the NotifyClientException class.:
`{"errors":[{"error":"ValidationError","message":"email_address Not a valid email address"}],"status_code":400}`

**Comment by gbishton** [comment]
I agree the string returned is difficult to parse and would prefer the JSON result suggested.
